src/documents/index.py

```diff
@@ -123,17 +124,17 @@ def open_index_searcher() -> Searcher:
         searcher.close()
 
 
-def update_document(writer: AsyncWriter, doc: Document):
-    tags = ",".join([t.name for t in doc.tags.all()])
-    tags_ids = ",".join([str(t.id) for t in doc.tags.all()])
-    notes = ",".join([str(c.note) for c in Note.objects.filter(document=doc)])
-    custom_fields = ",".join(
+def update_document(writer: AsyncWriter, doc: Document) -> None:
+    tags: str = ",".join([t.name for t in doc.tags.all()])
```

Comment:
Do these add anything?  It should be clear to any type checking tool these are `str`